From 6985e6eb8a1e831bc6d790b1e579c0313bcb6c15 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Fri, 12 Aug 2022 17:03:49 +0800
Subject: [PATCH 03/46] Revert "drm/i915: fix a possible refcount leak in
 intel_dp_add_mst_connector()"

This reverts commit 72f231b9a88abcfac9f5ddaa1a0aacb3f9f87ba5.
---
 drivers/gpu/drm/i915/display/intel_dp_mst.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/display/intel_dp_mst.c b/drivers/gpu/drm/i915/display/intel_dp_mst.c
index c59b43fb34a8..74d45a0eecb8 100644
--- a/drivers/gpu/drm/i915/display/intel_dp_mst.c
+++ b/drivers/gpu/drm/i915/display/intel_dp_mst.c
@@ -520,7 +520,6 @@ static struct drm_connector *intel_dp_add_mst_connector(struct drm_dp_mst_topolo
 	ret = drm_connector_init(dev, connector, &intel_dp_mst_connector_funcs,
 				 DRM_MODE_CONNECTOR_DisplayPort);
 	if (ret) {
-		drm_dp_mst_put_port_malloc(port);
 		intel_connector_free(intel_connector);
 		return NULL;
 	}
-- 
2.27.0

